File: samples/snippets/csharp/VS_Snippets_Data/XmlWriterSettings.NewLineOnAttributes/CS/writenewlineattrs.cs
using System;
using System.IO;
using System.Xml;

public class Sample {

  public static void Main() {
  
    XmlWriter writer = null;

    try {
//<snippet1>
XmlWriterSettings settings = new XmlWriterSettings();
settings.Indent = true;
settings.OmitXmlDeclaration = true;
settings.NewLineOnAttributes = true;
       
writer = XmlWriter.Create(Console.Out, settings);

writer.WriteStartElement("order");
writer.WriteAttributeString("orderID", "367A54");
writer.WriteAttributeString("date", "2001-05-03");
writer.WriteElementString("price", "19.95");
writer.WriteEndElement();
	
writer.Flush();
//</snippet1>
     } 
     finally  {
        if (writer != null)
          writer.Close();
     }
  } 
} 

File: samples/snippets/visualbasic/VS_Snippets_Data/XmlWriterSettings.NewLineOnAttributes/VB/writenewlineattrs.vb
Imports System.IO
Imports System.Xml

Public Class Sample 

    Public Shared Sub Main() 
  
        Dim writer As XmlWriter = Nothing

        Try 
            '<snippet1>
            Dim settings As New XmlWriterSettings()
            settings.Indent = True
            settings.OmitXmlDeclaration = True
            settings.NewLineOnAttributes = True
       
            writer = XmlWriter.Create(Console.Out, settings)

            writer.WriteStartElement("order")
            writer.WriteAttributeString("orderID", "367A54")
            writer.WriteAttributeString("date", "2001-05-03")
            writer.WriteElementString("price", "19.95")
            writer.WriteEndElement()
	
            writer.Flush()
            '</snippet1>
     
        Finally
           If writer IsNot Nothing Then
                writer.Close()
           End If
        End Try

    End Sub 
End Class 